### ISSUE TITLE
Strange error

### ISSUE THREAD

**Issue by ghost** [GordStephen/pdsite#3]
I get
```
$ pdsite build
Clearing old build folder... done.
Building site...pandoc: Could not parse YAML header: did not find expected key "source" (line 46, column 22)
pandoc: Could not parse YAML header: did not find expected key "source" (line 3, column 19)
 done.
[xan@argonia prova]$
```

**Comment by GordStephen** [comment]
That is indeed strange... I'll try to look at this when I have some free time next week. What version of Pandoc are you using?

Edit: May also be relevant: http://stackoverflow.com/questions/19520463/pandoc-cannot-parse-yaml-header-when-converting-md-to-pdf?rq=1 Can you post a minimal example of the document that fails to convert?

**Comment by hoijui** [comment]
yeah I think that was it. In markdown, to get a horizontal line, there should always be an empty line before and after the line with `---`.
